app/src/main/java/com/xmartlabs/daydreaming/model/Video.java

```diff
@@ -0,0 +1,24 @@
+package com.xmartlabs.daydreaming.model;
+
+import org.parceler.Parcel;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Created by chaca on 4/21/17.
+ */
+@Builder
+@Data
+public class Video {
+  String id;
+  String theme;
+  String type;
+
+  public String getId() {
```

Comment:
Maybe it was failing in the IDE but in runtime it was working. In that case, it's a plugin error.